test/functional/apps/dashboard/group5/embed_mode.ts

```diff
@@ -36,40 +42,88 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       });
       await PageObjects.common.navigateToApp('dashboard');
       await PageObjects.dashboard.preserveCrossAppState();
-      await PageObjects.dashboard.loadSavedDashboard('few panels');
+      await PageObjects.dashboard.loadSavedDashboard('dashboard with everything');
+
+      await browser.setWindowSize(1300, 900);
     });
 
-    it('hides the chrome', async () => {
-      const globalNavShown = await globalNav.exists();
-      expect(globalNavShown).to.be(true);
+    describe('default URL params', () => {
+      it('hides the chrome', async () => {
+        const globalNavShown = await globalNav.exists();
+        expect(globalNavShown).to.be(true);
 
-      const currentUrl = await browser.getCurrentUrl();
-      const newUrl = currentUrl + '&embed=true';
-      // Embed parameter only works on a hard refresh.
-      const useTimeStamp = true;
-      await browser.get(newUrl.toString(), useTimeStamp);
+        const currentUrl = await browser.getCurrentUrl();
+        const newUrl = currentUrl + '&embed=true';
+        // Embed parameter only works on a hard refresh.
+        const useTimeStamp = true;
+        await browser.get(newUrl.toString(), useTimeStamp);
+
+        await retry.try(async () => {
+          const globalNavHidden = !(await globalNav.exists());
+          expect(globalNavHidden).to.be(true);
+        });
+      });
+
+      it('expected elements are rendered and/or hidden', async () => {
+        await testSubjects.missingOrFail('top-nav');
+        await testSubjects.missingOrFail('queryInput');
+        await testSubjects.missingOrFail('superDatePickerToggleQuickMenuButton');
+        await testSubjects.existOrFail('globalQueryBar');
+      });
 
-      await retry.try(async () => {
-        const globalNavHidden = !(await globalNav.exists());
-        expect(globalNavHidden).to.be(true);
+      /**
+       * Skipping all render tests for now - there is a problem where the locally generated screenshots do not align with the
+       * CI screenshots due to (possibly) pixel density or something similar. This fix is super important to get in so we will
+       * have to resolve the issue with these new tests *after* FF for 8.9/8.8.2
```

Comment:
Merging this now but prioritizing unskipping these tests sounds good to me! We should open an issue to track.

Reply:
Opened [this issue](https://github.com/elastic/kibana/issues/160064) to track :+1: